tests/testEngineFactoryLocal.py

```diff
@@ -494,13 +494,13 @@ def testLoadSqlScriptNoDb(self):
         f.close()
         conn = getEngineFromFile(self.CREDFILE+".ini").connect()
         loadSqlScript(fN, username=self._user, host=self._host, port=self._port)
-        assert(10 == conn.execute("select sum(i) from %s.t" % self._dbA).first()[0])
+        assertEqual(10, conn.execute("select sum(i) from %s.t" % self._dbA).first()[0])
         utils.dropDb(conn, self._dbA)
         conn.close()
         os.remove(fN)
 
     def testLoadSqlScriptWithDb(self):
-        f, fN = tempfile.mkstemp(suffix=".csv", dir="/tmp", text="True")
+        f, fN = tempfile.mkstemp(suffix=".csv", True)
```

Comment:
Yes, I am rushing too much, I am testing while having this in review, I wanted to include it in the release. I found a couple of other small issues that I just fixed and pushed, now all tests pass cleanly